src/winetricks

```diff
@@ -16066,21 +16066,48 @@ load_origin()
         w_call corefonts
     fi
 
+    if w_workaround_wine_bug 44258 "Origin crashes on start."; then
+        w_override_app_dlls igoproxy.exe disabled d3d10
+        w_override_app_dlls igoproxy.exe disabled d3d10_1
+        w_override_app_dlls igoproxy.exe disabled d3d10core
+        w_override_app_dlls igoproxy.exe disabled d3d11
+        w_override_app_dlls igoproxy.exe disabled d3d12
+        w_override_app_dlls igoproxy.exe disabled dxgi
+        w_override_app_dlls igoproxy.exe disabled vulkan-1
+        w_override_app_dlls igoproxy.exe disabled winevulkan
+
+        if [ "${W_ARCH}" = "win64" ]; then
+            w_override_app_dlls igoproxy64.exe disabled d3d10
+            w_override_app_dlls igoproxy64.exe disabled d3d10_1
+            w_override_app_dlls igoproxy64.exe disabled d3d10core
+            w_override_app_dlls igoproxy64.exe disabled d3d11
+            w_override_app_dlls igoproxy64.exe disabled d3d12
+            w_override_app_dlls igoproxy64.exe disabled dxgi
+            w_override_app_dlls igoproxy64.exe disabled vulkan-1
+            w_override_app_dlls igoproxy64.exe disabled winevulkan
+        fi
+    fi
+
     if w_workaround_wine_bug 44985 "Disabling libglesv2 to make Store and Library function correctly."; then
         w_override_dlls disabled libglesv2
     fi
 
+    # Avoids "An unexpected error has occurred. Please try again in a few moments. Error: 327684:3"
+    # Games won't registor correctly unless disabled
+    if w_workaround_wine_bug 52781 "Origin does not notice games exiting, does not allow them to be relaunched."; then
+        w_override_app_dlls origin.exe disabled gameux
+    fi
+
     # Origin requirements
     w_call vcrun2010
+    w_call vcrun2013
```

Comment:
Is there a WineHQ bug for that (or the other vcrun verbs)?

Reply:
I don't remember seeing any, however the origin client it's self installs vcrun2010/vcrun2013/vcrun2015, I'd swapped out vcrun1015 with vcrun2019 for the newer dlls.

I've not tested how well Origin runs with the builtin dlls but the Winehq appdatabase Origin section does say to install these verbs, different origin components load these relevant dlls.